cddl/tst-info.cddl

```diff
@@ -21,11 +21,11 @@ MessageImprint = [
 
 profiled-etime = #6.1001(timeMap)
 timeMap = {
-  1 => int / float ; TIME_T
+  1 => ~time
   ? -8 => profiled-duration
   * int => any
 }
 profiled-duration = {* int => any}
 
-GeneralName = ( GeneralNameType : int, GeneralNameValue : any )
-SingleEntryGeneralNames = [ 1*1 GeneralName ]
+GeneralNameAttr = ( GeneralNameType : int, GeneralNameValue : any )
+GeneralName = [ + GeneralNameAttr ]
```

Comment:
hmm, this is not what 3161 says though: `tsa` is `GeneralName` singular, not `GeneralNames` plural...

Reply:
Yes.  that is what the PR says?

Reply:
Now I see what you were trying to do... RFC 5280...

Reply:
Fixed.

Reply:
https://github.com/ietf-rats/draft-birkholz-rats-epoch-marker/pull/20

Reply:
thanks!